README.md

```diff
@@ -1,43 +1,47 @@
 # Prism
 
-<a href="https://codeclimate.com/github/stoplightio/prism/maintainability"><img src="https://api.codeclimate.com/v1/badges/f5e363a7eb5b8f4e570f/maintainability" /></a>
-
 <a href="https://codeclimate.com/github/stoplightio/prism/test_coverage"><img src="https://api.codeclimate.com/v1/badges/f5e363a7eb5b8f4e570f/test_coverage" /></a>
 
 Prism is a set of packages that, given an API description, can:
 
 1. Spin up a mock HTTP server and respond realistically based on your requests
 1. Validate requests passing through against the provided API description
 
+**Note: this branch refers to Prism 3.x, which is the current version most likely you will use. If you're looking for the 2.x version, point your browser to the [right branch][2.x]**
+
 Being based on [Graphite], Prism supports any description format that Graphite supports:
 
 - OpenAPI v3.0
 - OpenAPI v2.0 (formerly Swagger)
 
 Prims is a multi-package repository:
 
-- `core:` basic interfaces and abstraction for API descriptions
-- `http:` A Prism implementation to work with HTTP Requests
-- `http-server:` A _[Fastify]_ instance that uses Prism to validate/mock/respond and forward to http requests
-- `cli:` A CLI to spin up servers locally easily
+- [`core:`][core] basic interfaces and abstraction for API descriptions
+- [`http:`][http] A Prism implementation to work with HTTP Requests
+- [`http-server:`][http-server] A _[Fastify]_ instance that uses Prism to validate/mock/respond and forward to http requests
+- [`cli:`][cli] A CLI to spin up servers locally easily
 
 Look at the relative repositories' README for the specific documentation.
 
 ## Install
 
-Prism is a Node module, and can be installed via NPM or Yarn:
+Most of the users will probably want to use the CLI, which is a Node module, and can either be installed via NPM or Yarn…
 
 ```bash
-npm install -g prism
+npm install -g @stoplight/prism-cli
 # or
-yarn global add prism
+yarn global add @stoplight/prism-cli
 ```
 
-_*TODO:* Create an executable which will run without needing to install a node module._
+…or if you do not want to install [Node](https://nodejs.org), you can download the latest release from [GitHub directly][download-release]
 
 ## Usage
 
-### Mock server
+### CLI
+
+We'll present here only the main use cases. For a complete overview of the CLI, you can consult the relevand [documentation][cli-docs]
+
+#### Mock server
 
 Running Prism on the CLI will create a HTTP mock server.
 
```

Comment:
I'd recommend using URLs for OAS documents, because the average user won't have cloned the repo when they are testing it out.

Reply:
Fantastic idea! Done!